apps/src/studio/blocks.js

```diff
@@ -3208,9 +3208,9 @@ exports.install = function (blockly, blockInstallOptions) {
       this.id +
       "', " +
       (this.getFieldValue('SPRITE') || '0') +
-      ", '" +
-      (this.getFieldValue('VALUE') || ' ') +
-      "');\n"
```

Comment:
Would this be more readable?
```
  generator.studio_saySpriteChoices = function () {
    const sprite = this.getFieldValue('SPRITE') || '0';
    const value = JSON.stringify(String(this.getFieldValue('VALUE') || ' '));
    // Generate JavaScript for saying (choices version).
    return `Studio.saySprite('block_id_${this.id}', '${sprite}', '${value}');\n`;
  };
```

Reply:
The `value` would have the quotes already. The fix would require, perhaps confusingly, but no more than the original change: 

```
  generator.studio_saySpriteChoices = function () {
    const sprite = this.getFieldValue('SPRITE') || '0';
    const value = JSON.stringify(String(this.getFieldValue('VALUE') || ' '));
    // Generate JavaScript for saying (choices version).
    return `Studio.saySprite('block_id_${this.id}', '${sprite}', ${value});\n`;
  };
```